src/errorHandle/switchErrors.ts

```diff
@@ -0,0 +1,15 @@
+import { VideoMetadataError } from "../search/videoMetadata";
+
+export function handleVideoMetadataError(error: VideoMetadataError): void {
```

Comment:
Just to give more context, this is where in a real case scenario you will have the logger of your app, as you can see, this is more visible and manageable as you can set severity and specify other fields for the logger